tool/site/server/src/services/session.ts

```diff
@@ -52,3 +54,28 @@ export function removeFromSessionValue(key: string, value: unknown, request: Hap
 export function clearSessionValue(key: string, request: Hapi.Request): void {
   request.yar.clear(key)
 }
+
+export function createSession(tokenResponse: Token, request: Hapi.Request, h: Hapi.ResponseToolkit): void {
+  request.cookieAuth.set({})
+  const accessTokenFetchTime = getUtcEpochSeconds()
+  const refreshTokenTimeout = 3599 / 3
```

Comment:
why are we using this value here as opposed to 1200 or 1199?

Reply:
3599 is the default for refresh token expiry, so in future we can use the full expiry, but will need some investigation to how to consume the refresh api on the oauth client library, something is mising from data being passed to it which only allows 1 refresh atm